app/pages/project/affinity/AffinityPage.tsx

```diff
@@ -98,8 +103,54 @@ const staticCols = [
 
 export default function AffinityPage() {
   const { project } = useProjectSelector()
-  const columns = useMemo(
-    () => [
+  const { data } = usePrefetchedQuery(antiAffinityGroupList({ project }).optionsFn())
+
+  const { mutateAsync: deleteGroup } = useApiMutation('antiAffinityGroupDelete', {
+    onSuccess(_data, variables) {
+      queryClient.invalidateEndpoint('antiAffinityGroupList')
+      addToast(
+        <>
+          Anti-affinity group <HL>{variables.path.antiAffinityGroup}</HL> deleted
+        </>
+      )
+    },
+  })
+
+  const makeActions = useCallback(
+    (antiAffinityGroup: AntiAffinityGroup): MenuAction[] => [
+      {
+        label: 'Delete',
+        onActivate() {
+          confirmAction({
+            actionType: 'danger',
+            doAction: () =>
+              deleteGroup({
+                path: { antiAffinityGroup: antiAffinityGroup.name },
+                query: { project },
+              }),
+            modalTitle: 'Delete anti-affinity group',
+            modalContent: (
+              <>
+                <p>
+                  Are you sure you want to delete the anti-affinity group{' '}
+                  <HL>{antiAffinityGroup.name}</HL>?
+                </p>
+                <p>
+                  Future placement of the affinity group’s members will not attempt to
+                  satisfy the affinity rules.
+                </p>
```

Comment:
I think we could probably just leave the line off, but if we keep it, the direction I'd go is something like:

> Member instances will no longer be kept apart in future placements, including starts or migrations.

One downside of this phrasing is that they could conceivably be in another group that also keeps them apart. So technically we can't say they won't be kept apart, only that this group is no longer keeping them apart.

> Member instances will no longer be kept apart in future placements unless they are in another anti-affinity group.

This is a bit silly. The use of "placements" I think is ok assuming we will have copy elsewhere (and docs, of course) explaining it, for example in the add instance form and/or the group create form.